### PR TITLE
added WIS

### PR DESCRIPTION
**Pull request by hightower8083**

### CODE REVIEW

File: data/picongpu-community.geojson [hunk 1]
```diff
@@ -346,11 +346,29 @@
       "geometry": {
         "type": "Point",
         "coordinates": [
-          -97.73634,
-          30.28892          
+          34.8083,
```
Comment (ax3l) [inline data/picongpu-community.geojson]: Please don't change Austin, Texas as well :)

**Review by ax3l — COMMENTED** [review]
Thank you for the update :sparkles: :rocket: 

Just a minor inline comment about an other entry that should not be changed as well :)

File: data/picongpu-community.geojson [hunk 2]
```diff
@@ -346,11 +346,29 @@
...
-    },    
+    },
+    {
+      "type": "Feature",
+      "properties": {
+          "name": "Weizmann Institute of Sience",
+          "contacts": [
+              "Dr. Igor Andriyash"
+          ],
+          "domain": ["physics", "laser","plasma"],
```
Comment (ax3l) [inline data/picongpu-community.geojson]: extra space after second `,` - if you like

File: data/picongpu-community.geojson [hunk 3]
```diff
@@ -346,11 +346,29 @@
...
+          34.8083,
+          31.9045         
         ]
       }
-    },    
+    },
+    {
+      "type": "Feature",
+      "properties": {
+          "name": "Weizmann Institute of Sience",
```
Comment (ax3l) [inline data/picongpu-community.geojson]: typo: science :)
```suggestion
          "name": "Weizmann Institute of Science",
```

File: data/picongpu-community.geojson (hunk 2 above)
Comment (ax3l) [inline data/picongpu-community.geojson]: extra space after second `,` - if you like
```suggestion
          "domain": ["physics", "laser", "plasma"],
```

File: data/picongpu-community.geojson (hunk 1 above)
Comment (ax3l) [inline data/picongpu-community.geojson]: Please don't change Austin, Texas as well :)
```suggestion
          -97.73634,
```

File: data/picongpu-community.geojson [hunk 4]
```diff
@@ -346,11 +346,29 @@
       "geometry": {
         "type": "Point",
         "coordinates": [
-          -97.73634,
-          30.28892          
+          34.8083,
+          31.9045
```
Comment (ax3l) [inline data/picongpu-community.geojson]:
```suggestion
          30.28892
```

**Comment by hightower8083** [comment]
thanx for corrections..! need a facepalm emojie here..

**Comment by ax3l** [comment]
Can you please "apply" my changes? :)
(There should be a button somewhere.)

**Comment by hightower8083** [comment]
can it merge now ?

File: data/picongpu-community.geojson (hunk 4 above)
Comment (ax3l) [reply data/picongpu-community.geojson]: pls that one as well :)

**Comment by hightower8083** [comment]
now i'm completely lost.. can you see any preview to know its looks ok ?
i don't get how it happened that i've modified texas..

**Review by ax3l — APPROVED** [review]

**Comment by ax3l** [comment]
All cool now, just some end-of-line whitespaces are now removed:
https://github.com/ComputationalRadiationPhysics/picongpu-communitymap/pull/5/files

**Pull request merged**